server/command/command.go

```diff
@@ -0,0 +1,102 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+
+	"github.com/mattermost/mattermost-plugin-cloudapps/server/configurator"
+	"github.com/mattermost/mattermost-plugin-cloudapps/server/constants"
+	"github.com/mattermost/mattermost-plugin-cloudapps/server/utils/md"
+)
+
+type Mattermost interface {
+	RegisterCommand(*model.Command) error
+}
+
+type Command interface {
+	Init(*configurator.BuildConfig) error
+	Handle(pluginContext *plugin.Context, commandArgs *model.CommandArgs) (out md.MD, err error)
+}
+
+type command struct {
+	Mattermost
+	configurator.Configurator
+}
```

Comment:
Is this responsible only for current slash commands or other commands as well? Could we have better naming?
If it's just a slash command I'd argue that structure should be more detailed, with subcommands and etc. To combine autocomplete and executing of the slash commands in the same code.

Reply:
Correct, just the `apps` command itself (install, list, subscribe, etc).

I am working on base structs for Forms, Modals, and Autocomplete now, under `./cloudapps`. The naming will hopefully become clearer then.

Agree on the parsing etc - leaving it for a future enhancement PR, the goal here is just to get the `Hello` going.